src/librbd/image/SetSnapRequest.cc

```diff
@@ -70,7 +68,7 @@ void SetSnapRequest<I>::send_init_exclusive_lock() {
   CephContext *cct = m_image_ctx.cct;
   ldout(cct, 10) << __func__ << dendl;
 
-  m_exclusive_lock = ExclusiveLock<I>::create(m_image_ctx);
```

Comment:
Nit: almost prefer the we drop ```ImageCtx::create_exclusive_lock``` and "exclusively" use the ```ExclusiveLock<I>::create``` form (since that's what we are now doing for all new code)